front/src/styles/variables.css

```diff
@@ -3,7 +3,10 @@
     --main-txt-color: white;
     --auxiliary-bg-color:  #FFA21C;
     --footer-bg-color:  #5956FF;
+    --serial-name-color: #292C33;
```

Comment:
is it just for serial name color?i see bellow the same color but for celebrity name